java/scala-logback/src/main/scala/HelloWorld.scala

```diff
@@ -0,0 +1,16 @@
+import io.sentry.Sentry
+
+object Application {
+  def main(args: Array[String]): Unit = {
+    Sentry.init("YOUR DSN")
```

Comment:
This works, but it's not the way most users will want to configure it in production. We may want to recommend they use it the same way we do Java: https://github.com/getsentry/examples/tree/master/java/logback

So roughly `SENTRY_DSN=x sbt run` or something?

Reply:
Gotcha. I have modified it so that the main function will take the DSN as the first argument. You would execute it like:
` sbt "run https://public:private@host:port/1"`

Reply:
You can also leave the entire `init` line out if you want, the Sentry SDK is built to automatically look for `SENTRY_DSN` environment or `-Dsentry.dsn` JVM property. Totally up to you though.

Reply:
I see. That makes sense. I've confirmed that that works and pushed up the changes.